src/domain/community/virtualContributor/vcProfilePage/VCProfileContentView.tsx

```diff
@@ -154,3 +202,34 @@ export const VCProfileContentView = ({
 };
 
 export default VCProfileContentView;
+
+function SectionTitle({ children }) {
+  return (
+    <BlockTitle display={'flex'} alignItems={'center'} gap={gutters(0.5)}>
+      {children}
+    </BlockTitle>
+  );
+}
+
+function SectionWrapper({ children }: PropsWithChildren<{}>) {
+  return (
+    <Gutters disablePadding sx={{ flexDirection: 'row', flexWrap: 'wrap' }}>
+      {children}
+    </Gutters>
+  );
+}
+
+function SectionItem({ children }: PropsWithChildren<{}>) {
```

Comment:
you can just use `PropsWithChildren` with no `<>` and it will default to `unknown`